internal/oci/runtime_oci.go

```diff
@@ -1294,13 +1295,32 @@ func (r *runtimeOCI) AttachContainer(ctx context.Context, c *Container, inputStr
 
 	defer controlFile.Close()
 
+	var (
+		lastSize         remotecommand.TerminalSize
+		controlFileLock  sync.Mutex
+		controlFileValid = true
```

Comment:
it doesn't seem like this ever gets set back to true. if we hit this issue then we'll not get anoter resize event, which doesn't seem right.

It seems the investigation is pointing to the fact that the buffer is getting overloaded. if we see this at container startup with two events happening quikcly, maybe we just double the buffer size?

I don't think this is the right approach...

Reply:
Yeah, you are right. I'm going to revert this change. The actual fix should be in conmon. https://github.com/containers/conmon/pull/566